### PR TITLE
Bug/polling not working

### LINKED ISSUE #194: Poll commands not working
- Having a service provisioned in iota-json:

> Method: post
> Url: http://192.168.137.16:8086/iot/services
> Headers: {'Fiware-Service': u'iotaf006011www', 'Content-Type': 'application/json', 'Fiware-ServicePath': '/iotassf0601www', 'X-Auth-Token': '8d75a8d3b5a3463b95a1872837054c7d'}
> Payload sent: '{
  "services": [
    {
      "apikey": "a00601www001",
      "protocol": [
        "IoTA-JSON"
      ],
      "entity_type": "device",
      "cbroker": "http://10.0.0.2:1026"
    }
  ]
}'

- And a device provisioned with poll command:

> Method: post
> Url: http://192.168.137.16:8086/iot/devices
> Headers: {'Fiware-Service': 'iotaf006011www', 'Content-Type': 'application/json', 'Fiware-ServicePath': '/iotassf0601www', 'X-Auth-Token': '8d75a8d3b5a3463b95a1872837054c7d'}
> Payload sent: '{
  "devices": [
    {
      "commands": [
        {
          "type": "command",
          "name": "ping",
          "value": ""
        }
      ],
      "protocol": "IoTA-JSON",
      "entity_name": "diota00601www01",
      "entity_type": "device",
      "static_attributes": [
        {
          "type": "type1",
          "name": "name1",
          "value": "value1"
        }
      ],
      "attributes": [
        {
          "type": "percent",
          "name": "humidity",
          "object_id": "hum"
        }
      ],
      "device_id": "diota00601www01"
    }
  ]
}'

- When sending a command from CB to iota-json

> Url: http://192.168.137.1:1026/v1/contextEntities/type/device/id/diota00601www01
> Method: put
> Headers: {'Fiware-Service': u'iotaf006011www', 'content-type': 'application/json', 'Fiware-Servicepath': u'/iotassf0601www', 'Accept': 'application/json', 'x-auth-token': '8d75a8d3b5a3463b95a1872837054c7d'}
> Payload sent: '{
  "attributes": [
    {
      "type": "command",
      "name": "ping",
      "value": "21"
    }
  ]
}'

The iota does not respond and appears the following ERROR in the log:

time=2016-10-28T09:32:04.253Z | lvl=ERROR | corr=62f0e97e-9cf1-11e6-909a-fa163e433687 | trans=24c74794-be0a-4421-8ed1-d835b88284e2 | op=IoTAgentNGSI.MongoDBGroupRegister | srv=iotaf006021www | subsrv=/iotassf0602www | msg=TypeError: Cannot read property 'statusCode' of undefined | comp=IoTAgent

### PR DESCRIPTION
**Pull request by dmoranj**
Fixes #194

### CODE REVIEW

**Comment by XavierVal** [comment]
LGTM

File: test/unit/commandsPolling-test.js [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        });
+    });
+
+    describe('When a device asks for the list of commands and there is more than one command', function() {
+        it('should retrieve the list sepparated by the "#" character');
+    });
+
+    describe('When a device sends the result for a pending command', function() {
+        var commandResponse = {
+            uri: 'http://localhost:' + config.http.port + '/iot/d/commands',
```
Comment (manucarrace) [inline test/unit/commandsPolling-test.js]: Is that url correct???

File: test/unit/commandsPolling-test.js [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+                    list.commands[0].name.should.equal('PING');
+                    done();
+                });
+            });
+        });
+    });
+
+    describe('When a device asks for the pending commands', function() {
+        var deviceRequest = {
+            url: 'http://localhost:' + config.http.port + '/iot/d',
```
Comment (manucarrace) [inline test/unit/commandsPolling-test.js]: I expect /iot/json

File: lib/constants.js [hunk 3]
```diff
@@ -34,6 +34,7 @@ module.exports = {
 
     HTTP_MEASURE_PATH: '/iot/d',
```
Comment (manucarrace) [inline lib/constants.js]: ???

**Pull request merged**